Allow ignoring fields (#34)
rogiervandergeer/pydargs: README.md

  will be parsed into the first type that returns a valid result. Note that this means
  that `str | int` will _always_ result in a value of type `str`.
- Any other type that can be instantiated from a string, such as `Path`.

## Metadata

### Ignoring fields
Fields can be ignored by adding the `ignore_arg` metadata field:
```python
@dataclass
class Config:
    number: int
    ignored: str = field(metadata=dict(ignore_arg=True))
```

> When indicated, this field is not added to the parser and cannot be overridden via the cli.

Reviewer: ```suggestion
When indicated, this field is not added to the parser and cannot be overridden with an argument.
```
